utils/check_dashboard_name_uniqueness.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+const glob = require('glob');
+const {
+  readQuickstartFile,
+  removeRepoPathPrefix,
+  findMainQuickstartConfigFiles,
+} = require('./helpers');
+
+const getQuickstartDashboardConfigs = (quickstartConfigPath) => {
+  const splitConfigPath = quickstartConfigPath.split('/');
+  splitConfigPath.pop();
```

Comment:
why do you want to remove the last element here?

Reply:
So this removes the initial config file that is located in the main part of the quickstart 

`[
  '',
  'Users',
  'lbrammer',
  'Work',
  'dev-exp',
  'newrelic-quickstarts',
  'quickstarts',
  'wordpress',
  'config.yml'
]`

This allows us to find any quickstarts that have a dashboard with a json file within the dashboard directory of a quickstart via 

`const globPattern = `${splitConfigPath.join('/')}/dashboards/*.+(json)`;

  return glob.sync(globPattern);`

Reply:
my recommendation would be to create a separate util so we only have to do the path search once! 

```js

/**
 * Finds the path to all dashboard config jsons
 * @returns {String[]} An array of the file paths
 */
const findQuickstartDashboardJsons = () =>
  glob.sync(
    path.resolve(process.cwd(), '../quickstarts/**/dashboards/*.+(json)')
  );
```

Reply:
Done :) this is a good replacement to just get it from the start. thanks!